template.md

```diff
@@ -18,15 +18,45 @@ Explain the proposal as if it was already included in the engine and you were te
 - If applicable, provide sample error messages, deprecation warnings, or migration guidance.
 - If applicable, explain how this feature compares to similar existing features, and in what situations the user would use each one.
 
-## Reference-level explanation
+## \[Optional\] Examples
 
-This is the technical portion of the RFC. Explain the design in sufficient detail that:
+Lay out the examples that Bevy should add to either the examples folders or the doc comments to help new users discover and learn about this feature.
+These can either be completely new, or simply changes to existing examples.
+When completing this section, you can but do not have to write all of these examples yourself.
+Describing them in sufficient detail that another developer can do so is perfectly fine.
+If your examples are particularly long, please provide a brief description and link to a [Gist](https://gist.github.com/) for each.
```

Comment:
What about suggesting to wrap the code into a collapsible? A gist could be accidentally deleted and difficult to track and be kept in sync with the RFC.

Reply:
I agree with this; I'd forgotten about this functionality! This looks to be a pretty good link that explains how to do so: https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab